pysat/tests/test_instrument.py

```diff
@@ -842,17 +842,17 @@ def test_custom_instrument_load_3(self):
     # -------------------------------------------------------------------------
     def test_basic_data_access_by_name(self):
         self.testInst.load(self.ref_time.year, self.ref_doy)
-        assert np.all(self.testInst['uts'] == self.testInst.data['uts'])
+        assert np.all(self.testInst['mlt'] == self.testInst.data['mlt'])
```

Comment:
Comparing for data values makes more sense here and below, as for some tests the 'uts' will be retrieved as different data types now that timestamps are utc.